app/classifier/classifier.py

```diff
@@ -269,12 +270,19 @@ def learn_all_symbols_together(self):
                 feature_vector = featureextractor.get_features(training_element)
                 feature_vectors.append(feature_vector)
                 results.append(sym)
-        knn_model = KNeighborsClassifier(n_neighbors=5).\
-            fit(feature_vectors, results)
-        file_with_model = \
-            open(Classifier._get_file_path(self.files[MODEL_FILE], ""), 'wb')
-        pickle.dump(knn_model, file_with_model)
-        file_with_model.close()
+        if self.symbol_list:
+            knn_model = KNeighborsClassifier(n_neighbors=5).\
```

Comment:
- [ ] How about moving the `MIN_TRAINIG_SIZE` constatnt from `app.py` to `classifier.py` and using it here like this: ` knn_model = KNeighborsClassifier(n_neighbors=MIN_TRAINIG_SIZE)`